### PR TITLE
Json buffer

### PR DESCRIPTION
**Pull request by Renaud2002**
@SantiM1 the added code I think should take care of the comma but needs to be tested to make sure.

### CODE REVIEW

File: src/main/webapp/script.js [hunk 1]
```diff
@@ -26,6 +26,11 @@ const mobileMenu = () => {
 //toggle the div of hamburger menu
 menu.addEventListener('click', mobileMenu);
 
+
+  // Add it to the page.
+  const greetingContainer = document.getElementById('greeting-container');
+  greetingContainer.innerText = greeting;
```
Comment (SantiM1) [inline src/main/webapp/script.js]: We should delete this piece of code, seems kinda random

File: src/main/webapp/script.js [hunk 2]
```diff
@@ -80,7 +87,18 @@ navLogo.addEventListener('click', hideMobileMenu)
...
+
+
+  const uploaded_images = await fetch('/list-images' , {
+    method: 'POST', // or 'PUT'
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(value),
+  });
+
```
Comment (SantiM1) [inline src/main/webapp/script.js:100]: Is this the Json Buffer? If so, looks good  👍

**Review by SantiM1 — CHANGES_REQUESTED** [review]
We could add the greetings somewhere but maybe delete it so it doesn't stay there accidentally

File: src/main/webapp/script.js (hunk 1 above)
Comment (Renaud2002) [reply src/main/webapp/script.js]: Yeah couldn't remember why it was there, it doesn't do anything it was from a copy and past I did, I will delete it.

File: src/main/webapp/script.js (hunk 2 above)
Comment (Renaud2002) [reply src/main/webapp/script.js:100]: Yeah I think thats it when we merge and test we will see if it actually works, or I might be mistaken.

File: src/main/webapp/script.js (hunk 1 above)
Comment (Renaud2002) [reply src/main/webapp/script.js]: @SantiM1 I actually I don't know if I was the one to put it seem to be in the code that Eric wrote so not sure, I can comment it for now and ask him later.

**Comment by Renaud2002** [comment]
@SantiM1 you can go ahead a review again had to pull some stuff from main after that we should be able to merge with main and test it out.

**Review by SantiM1 — APPROVED** [review]

**Pull request merged**